app.js

```diff
@@ -1,59 +1,99 @@
 'use strict'
+
 // store hours
 var hours =['6am: ', '7am: ', '8am: ', '9am: ', '10am: ', '11am: ', '12pm: ', '1pm: ', '2pm: ', '3pm: ', '4pm: ', '5pm: ', '6pm: ', '7pm: '];
+var pikeStore = new Stores ('Pike Place', 23, 65, 6.3);
+var seaTac = new Stores('SeaTac', 3, 24, 1.2);
+var seattleCenter = new Stores('Seattle Center', 11, 38, 3.7);
+var capHill = new Stores('Capital Hill', 20, 38, 2.3);
+var alki = new Stores('Alki', 2, 16, 4.6);
 
-//store names
-var nameStores = [pikeStore,seaTac,seattleCenter,capHill,alki];
 
 //store Objects
+function Stores (name, minCust, maxCust, avgCookie) {
+  this.name = name;
+  this.min = minCust;
+  this.max = maxCust;
+  this.avgCookie = avgCookie;
+  this.totalSales = 0;
+  this.hourSales = [];
+};
 
-function Store(name, min, max, avgCookie) {
-  newStore.name = name;
-  newStore.min = min;
-  newStore.max = max;
-  newStore.avgCookie = avgCookie;
-  newStore.totalSales = 0;
-  newStore.hourSales = [];
-  return newStore;
-
-}
+//Random Customer
 Stores.prototype.storeRandomCust = function () {
   return Math.random() * (this.maxCust - this.minCust) + this.minCust;
```

Comment:
Looks like the error is coming from the formula to generate a random number. This doesn't seem to actually generate a number. I don't think your stores have a maxCust and minCust property...